misc/install-deps.sh

```diff
@@ -14,20 +14,20 @@ distro=$(grep "^ID=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
 case $distro in
     "ubuntu" | "debian")
         apt-get $OPT install pandoc libdw-dev libpython2.7-dev libncursesw5-dev pkg-config
-        apt-get $OPT install libcapstone-dev libluajit-5.1-dev ;;
+        apt-get $OPT install libcapstone-dev libluajit-5.1-dev || true ;;
```

Comment:
Seems you're trying to suppress the error from the absence of the package `libcapstone-dev` which doesn't exist in Ubuntu 14.04 LTS. 

But with this approach, this will skip the installation of `libcapstone-dev` as well as `libluajit-5.1-dev`.

https://pkgs.org/download/libluajit-5.1-dev

Since luajit package exist in Ubuntu 14.04 LTS, how about moving `libluajit-5.1-dev` to upper line?